devlib/utils/gem5.py

```diff
@@ -39,5 +46,6 @@ def iter_statistics_dump(stats_file):
             if res:
                 k = res.group("key")
                 v = res.group("value").split()
+                v = map(best_effort_numeric, v)
```

Comment:
Hm, yeah, maybe a `TargetError` with something like "encountered non-numeric gem5 stat: \<blah>" would be appropriate here, and then let the client code figure out the best way of handling that...

@Sticklyman1936 , @avanlaer ?